Uzira.rte/Devices/Tools/Spade.ini

```diff
@@ -0,0 +1,9 @@
+///////////////////////////////////////////////////////////////////////
+// Spade
+
+
+AddDevice = HDFirearm
+	CopyOf = Shovel
```

Comment:
I don't think skeletons should be able to make sandbags lol

Oh, also, another thing I sometimes think about but never remember to mention, is that the sharpaim hit overhead for shovel works a lot less well for mouse. It's usable but a lot more fiddly!
Not sure what you can do about that, but wanted to mention it so you know at least (applies to gunsword as well, but less so since it's got sharpaim dots - maybe the solution is just to show sharpaim dots if not kbd-only controlled).

Reply:
I'm afraid it will have to wait for a "secondary action" key mapping

The sandbags though - I do agree with you, I didn't intend to give them that ability, but I don't want to make a carbon copy of the shovel without the sandbag feature either

Reply:
Hmm, there's no ini way to get rid of this. I guess you can either
1. Have a base shovel with no pie slice and have the ronin one and this one copyof it, and the ronin one add a pieslice
2. Have a script on this shovel that removes the pie slice
3. Something else I'm not thinking of

Reply:
What if instead I think of a justification for the skeleton sandbag use through lore?

Reply:
hahaha I suppose that's legal too